### ISSUE TITLE
Issue with moving inline comments

### ISSUE THREAD

**Issue by kentcdodds** [babel/babel#2068]
TL;DR: Here's [the issue](http://babeljs.io/repl/#?experimental=true&evaluate=true&loose=false&spec=false&playground=true&code=var%20x%20%3D%20%7B%0A%20%20injectable%3A%20%2F*%20%40ngInject%20*%2F%20function(someThing)%20%7B%0A%20%20%7D%0A%7D%3B):

Input:

```javascript
var x = {
  injectable: /* @ngInject */ function(someThing) {
  }
};
```

Output:

```javascript
"use strict";

var x = {
  injectable: function injectable(someThing) {}
};
/* @ngInject */
```

Because the `/* @ngInject */` comment is moved, one of my build tools doesn't annotate the function for me like it should.

If you're interested in the use case, read on...

I'm using babel to develop angular apps with ES6. One tool that angular developers use a lot to support angular's dependency injection is called [`ng-annotate`](https://github.com/olov/ng-annotate). Basically, because of the way angular's dependency injection works, you need to annotate your functions with a `$inject` property as an array of strings to indicate the dependencies. `ng-annotate` does this for you so you don't have to worry about keeping your function's arguments and the `$inject` array in sync.

`/* @ngInject */` is a queue to `ng-annotate` to annotate a particular function.

**Comment by hzoo** [comment]
I think this might be the same/similar to https://github.com/babel/babel/issues/1798?
